com.unity.formats.fbx/Editor/ExportModelSettings.cs

```diff
@@ -95,6 +95,15 @@ public override void OnInspectorGUI ()
             exportSettings.SetExportUnredererd(EditorGUILayout.Toggle(exportSettings.ExportUnrendered));
             EditorGUI.EndDisabledGroup ();
             GUILayout.EndHorizontal ();
+            
+            GUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField(new GUIContent("Preserve Import Settings for Existing FBX",
```

Comment:
This does not match the screenshot that's in the Conversation section of this PR (I only see "Preserve Import Settings"). Is it that the text is clipped because it is too long?

Reply:
Yeah, the full message is clipped in the actual editor menu.